torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -24,6 +24,7 @@ def mixup_one_hot_label(one_hot_label_batch: torch.Tensor, *, lam: float, inplac
 
 
 def cutmix_image(image: torch.Tensor, *, box: Tuple[int, int, int, int], inplace: bool = False) -> torch.Tensor:
+    # Shouldn't be we using `image_batch` similar to `image_batch`, to indicate that the input must be a batch?
```

Comment:
Yes, we can rename this. Note, that I also left out any input validation here for simplicity. Let me add a minimalistic one to help reviewers.

Reply:
Makes sense.